### PR TITLE
12.0 mig base import security group

### PR DESCRIPTION
**Pull request by ajayjaviya**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]
LGTM :+1:

**Comment by ivantodorovich** [comment]
@alejandrosantana and @espo-tony, as module contributors maybe you are interested in this :)
We'd appreciate a review

**Comment by ivantodorovich** [comment]
@i-vyshnevska @StefanRijnhart , as module contributors maybe you are interested in this :)
We'd appreciate a review

File: base_import_security_group/i18n/base_import_security_group.pot [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Translation of Odoo Server.
```
Comment (i-vyshnevska) [inline base_import_security_group/i18n/base_import_security_group.pot]: you can drop this file

**Review by i-vyshnevska — APPROVED** [review]
Code and functional test ok

File: base_import_security_group/i18n/base_import_security_group.pot (hunk 1 above)
Comment (ajayjaviya) [reply base_import_security_group/i18n/base_import_security_group.pot]: @i-vyshnevska : I have remove `base_import_security_group.pot` file

**Comment by ivantodorovich** [comment]
@ajayjaviya can you squash your commits?

**Review by StefanRijnhart — APPROVED** [review]
Very nice!

**Pull request merged**